package.json

```diff
@@ -39,12 +39,12 @@
     "src"
   ],
   "dependencies": {
-    "dragselect": "^2.5.4"
+    "dragselect": "^2.6.0"
   },
   "devDependencies": {
     "@blockly/dev-scripts": "^1.2.28",
-    "@blockly/dev-tools": "^5.0.2",
-    "blockly": "^9.0.0"
+    "@blockly/dev-tools": "^5.0.5",
+    "blockly": "^9.2.1"
```

Comment:
Is bumping the Blockly version necessary here? I feel like the change to dragselect should be sufficient and there's no reason to disallow early Blockly versions if the code is compatible.

Reply:
Hi! I admit that bumping the Blockly version is not something necessary, here I just want to make sure that the latest Block version works with this plugin, and it's only for development purpose. (`devDependencies` specifies packages that are only needed for local development and testing https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file ). I'm not bumping the blockly version in peerDependencies, so the change here is not related to disallow early Blockly versions.

Reply:
You can find the table here more helpful: https://www.geeksforgeeks.org/difference-between-dependencies-devdependencies-and-peerdependencies/